.github/workflows/ci.yaml

```diff
@@ -39,3 +39,17 @@ jobs:
         # Go 1.20. So we skip this step for Go 1.19.
         if: matrix.go-version != '1.19.x'
         run: make runconformance
+  slowtest:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+      - name: Install Go
+        uses: actions/setup-go@v4
+        with:
+          # only the latest
+          go-version: 1.21.x
+      - name: Run Slow Tests
+        run: make slowtest
```

Comment:
Talked to Josh live about this, but I'd love for this to be carved up a little differently. Locally, I'd really like `make` and `make test` to run these slower smoke tests. In CI, the highly-matrixed tests should run all the tests _except_ the smoke tests. The new job (`slowtest`) should run the slow tests, but it would be okay with me if it _also_ ran the faster tests.

I don't care that much whether we carve up the tests with an environment variable, `testing.Short()`, or with build tags.

Reply:
`make test` now runs both short and slow tests. CI "Test" job only runs short tests, and there's a separate "Slow Tests" job for the others.

Also, while factoring out slow tests to a parallel job, I did the same for conformance tests, to shorten the critical path through the CI testing.

Reply:
Because the main CI job must re-build the code (to instrument with race detector) and also does linting, the new shorttest job is actually faster than that other job (for now at least). So this new job does not increase the total duration. The longest job is still Windows, by a good margin.